Description: Autoinstall modules that were just added
---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: d8f198f4327a5f91c963041e7b326d9f4273fa62
Bug-Ubuntu: https://launchpad.net/bugs/867922
Forwarded: not-needed

--- dkms-2.2.0.2.orig/dkms
+++ dkms-2.2.0.2/dkms
@@ -3131,8 +3131,6 @@ autoinstall() {
     # Walk through our list of installed and built modules, and create
     # a list of modules that need to be reinstalled.
     while read status mvka; do
-        # if it has not been built, do not try and build it now.
-        [[ $status = added ]] && continue
         IFS='/' read m v k a <<< "$mvka"
         [[ ! $last_m ]] && last_m="$m"
         # If the module is already installed or weak-installed, skip it.
